src/definitions/modules/flyout.less

```diff
@@ -0,0 +1,850 @@
+  color: #FFFFFF;
+}
+
+/*******************************
+          Variations
+*******************************/
+
+/*--------------
+     Width
+---------------*/
+
+/* Left / Right */
+& when (@variationFlyoutLeft) or (@variationFlyoutRight) {
+  & when (@variationFlyoutThin) {
+    .ui.thin.left.flyout,
+    .ui.thin.right.flyout {
+      width: @thinWidth;
+    }
+
+    .ui[class*="very thin"].left.flyout,
+    .ui[class*="very thin"].right.flyout {
+      width: @veryThinWidth;
+    }
+  }
+
+  .ui.left.flyout,
+  .ui.right.flyout {
+    width: @width;
+  }
+
+  & when (@variationFlyoutWide) {
+    .ui.wide.left.flyout,
+    .ui.wide.right.flyout {
+      width: @wideWidth;
+    }
+
+    .ui[class*="very wide"].left.flyout,
+    .ui[class*="very wide"].right.flyout {
+      width: @veryWideWidth;
+    }
+  }
+
+  /* Left Visible */
+  & when (@variationFlyoutLeft) {
+    & when (@variationFlyoutThin) {
+      .ui.visible.thin.left.flyout ~ .fixed,
+      .ui.visible.thin.left.flyout ~ .pusher {
+        transform: translate3d(@thinWidth, 0, 0);
+      }
+
+      .ui.visible[class*="very thin"].left.flyout ~ .fixed,
+      .ui.visible[class*="very thin"].left.flyout ~ .pusher {
+        transform: translate3d(@veryThinWidth, 0, 0);
+      }
+    }
+
+    & when (@variationFlyoutWide) {
+      .ui.visible.wide.left.flyout ~ .fixed,
+      .ui.visible.wide.left.flyout ~ .pusher {
+        transform: translate3d(@wideWidth, 0, 0);
+      }
+
+      .ui.visible[class*="very wide"].left.flyout ~ .fixed,
+      .ui.visible[class*="very wide"].left.flyout ~ .pusher {
+        transform: translate3d(@veryWideWidth, 0, 0);
+      }
+    }
+  }
+
+  /* Right Visible */
+  & when (@variationFlyoutRight) {
+    & when (@variationFlyoutThin) {
+      .ui.visible.thin.right.flyout ~ .fixed,
+      .ui.visible.thin.right.flyout ~ .pusher {
+        transform: translate3d(-@thinWidth, 0, 0);
+      }
+
+      .ui.visible[class*="very thin"].right.flyout ~ .fixed,
+      .ui.visible[class*="very thin"].right.flyout ~ .pusher {
+        transform: translate3d(-@veryThinWidth, 0, 0);
+      }
+    }
+
+    & when (@variationFlyoutWide) {
+      .ui.visible.wide.right.flyout ~ .fixed,
+      .ui.visible.wide.right.flyout ~ .pusher {
+        transform: translate3d(-@wideWidth, 0, 0);
+      }
+
+      .ui.visible[class*="very wide"].right.flyout ~ .fixed,
+      .ui.visible[class*="very wide"].right.flyout ~ .pusher {
+        transform: translate3d(-@veryWideWidth, 0, 0);
+      }
+    }
+  }
+
+  /* Fullscreen */
+
+  .ui.fullscreen.left.flyout,
+  .ui.fullscreen.right.flyout {
+    width: 100% !important;
+  }
+
+
+  /*-------------------
+    Column Width
+  --------------------*/
+
+  & when (@variationFlyoutColumnWidth) {
+    
+    /* Sizing Combinations */
+
+    .ui[class*="one wide"].left.flyout:not(.fullscreen),
+    .ui[class*="one wide"].right.flyout:not(.fullscreen) {
+      width: @oneWide;
+    }
+
```

Comment:
It's to prevent an eventual user error:
![image](https://user-images.githubusercontent.com/7557689/172646043-7cc582fb-abcb-4d4d-981d-9723b0adfe28.png)

This can happen if user use `ui top eight wide flyout`, for example. But if you think it is unnecessary, i'll remove it.

Reply:
I still think we should remove/simply that. A `wide` flyout is supposed to hava a dedicated width (which does not really make sense if one also uses "top", but if one still combines it, then the screenshot is the expected result (the flyout is on top and is eight wide ;))

Reply:
Resolved by deb1a561dfc99c26494d0e5c503c1118c7d4ccd3.